news/4310.bugfix

```diff
@@ -0,0 +1,2 @@
+Works around UnicodeDecodeError in decoding console output using ``repr()`` on
```

Comment:
Apologies for my poor English; "UnicodeDecodeError using ``repr()`` in decoding console output" would make sense more than my committed message. If it is allowed and preferred, I will push one more commit to fix the words.